timesheets3/src/classes/timesheetsController.cls

```diff
@@ -0,0 +1,56 @@
+public with sharing class timesheetsController {
+
+            @AuraEnabled
+    public static List<Timesheet__c> getTimesheets(String view) {
+        
+        String[] fieldsToCheck = new String[] {
+            'Id','Name', 'Employee__c', 'Start_Date__c', 'End_Date__c', 
+            'Total_Hours__c', 'Gross_Pay__c','Taxes__c','Type__c','Submmited__c'
+        };
+        
+        Map<String,Schema.SObjectField> fieldDescribeTokens = 
+            Schema.SObjectType.Timesheet__c.fields.getMap();
+        
+        for(String field : fieldsToCheck) {
+            if( ! fieldDescribeTokens.get(field).getDescribe().isAccessible()) {
+                //throw new System.NoAccessException();
+                return null;
+            }
+        }
+        
+        System.debug('Estoy en el controlador');
+        
+        if(view == '1'){
```

Comment:
Using static SOQL is not the right thing to do here. You should use Dynamic SOQL instead of this different set of if-else-if statements.